### PR TITLE
T0 scenario for repacked pp

### PR DESCRIPTION
**Pull request by mandrenguyen**
New T0 scenario for pp reco, but with the RAW data format that is repacked at the HLT.
https://cms-talk.web.cern.ch/t/possible-issue-with-t0-config-for-hi-prompt-reco

A backport to 13_2_X will be made.

Tested with the help of German.

### CODE REVIEW

**Comment by mandrenguyen** [comment]
urgent 
For ongoing data-taking

**Comment by mandrenguyen** [comment]
please test

**Comment by mmusich** [comment]
I know this is urgent, but shouldn't it at least added in the tests:

https://github.com/cms-sw/cmssw/blob/56711f4be87ab0b56c9bcd9b703f9cb33a2bc6c9/Configuration/DataProcessing/test/run_CfgTest.sh#L60

?

File: Configuration/DataProcessing/python/Impl/ppEra_Run3_2023_repacked.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+"""
+_ppEra_Run3_2023_
+
+Scenario supporting proton collisions for 2023 with updated HCAL Barrel thresholds
```
Comment (mmusich) [inline Configuration/DataProcessing/python/Impl/ppEra_Run3_2023_repacked.py:5]:
```suggestion
_ppEra_Run3_2023_repacked_

Scenario supporting HIon collisions for 2023 with event content repacked at HLT
```

**Comment by mandrenguyen** [comment]
> I know this is urgent, but shouldn't it at least added in the tests:
> 
> https://github.com/cms-sw/cmssw/blob/56711f4be87ab0b56c9bcd9b703f9cb33a2bc6c9/Configuration/DataProcessing/test/run_CfgTest.sh#L60
> 
> ?
Ah, thanks, so just:
`declare -a arr=("ppEra_Run3" "ppEra_Run3_repacked")`
?

**Comment by mmusich** [comment]
>  declare -a arr=("ppEra_Run3" "ppEra_Run3_repacked")

I think so, but haven't tested it :)

File: Configuration/DataProcessing/test/run_CfgTest.sh [hunk 2]
```diff
@@ -57,7 +57,7 @@ do
      runTest "${SCRAM_TEST_PATH}/RunPromptReco.py --scenario $scenario --reco --aod --dqmio --global-tag GLOBALTAG --lfn=/store/whatever  --alcareco TkAlMinBias+SiStripCalMinBias --PhysicsSkim=@SingleMuon"
 done
 
-declare -a arr=("ppEra_Run3")
+declare -a arr=("ppEra_Run3" "ppEra_Run3_repacked")
```
Comment (slava77) [inline Configuration/DataProcessing/test/run_CfgTest.sh:60]: isn't `_2023_` missing? (also for the already present)

**Comment by mandrenguyen** [comment]
please test

**Comment by rappoccio** [comment]
+1

**Pull request merged**